browser.js

```diff
@@ -0,0 +1,11 @@
+var EventEmitter = require('events').EventEmitter
+var getBrowserRTC = require('get-browser-rtc')
+
+module.exports = function (opts) {
+  var wrtc = new EventEmitter()
+  // If electron-webrtc is required in the browser, return the browser RTC implementation.
+  return Object.assign(wrtc, getBrowserRTC(), {
```

Comment:
If you insist, I'll add it. But I don't think it's a big deal. Browsers that support webrtc all support `Object.assign`.

The only browser this excludes is IE11 and below. v